doc/theory/glossary.tex

```diff
@@ -54,7 +210,7 @@
 	d_i^\pm I^N = I_{a_1} \times \cdots \times d^\pm I_{a_{k_i}} \times \cdots \times I_{a_{N.}}
```

Comment:
Should "I_{a_{N.}}" be "I_{a_{N}}."?

Reply:
Yeah, that's a decision we need to make. I find this way looks better. If you prefer the other I will change it. I just wanted you to know that it is intentionally and consistently done. The rule being: If a displayed equation ends in a period after a subscript, the period appears in the subscript.

Reply:
Just saw the output online and in my view the dots are much too small (and too low). Thus, I'd prefer them to be outside the subscripts throughout.